### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by chepkok3**
[ ] Created the database based on a diagram

WhatsApp Image 2023-04-12 at 17 54 05
- The patients table has a primary key column id which is referenced as a foreign key patient_id in the medical_histories table, establishing a one-to-many relationship between patients and medical_histories.
- The medical_histories table has a primary key column id and is also referenced as a foreign key medical_history_id in the invoices and joined tables, establishing a one-to-many relationship between medical_histories and invoices, as well as a many-to-many relationship between medical_histories and treatments through the joined table.
- The invoices table has a primary key column id and is referenced as a foreign key invoice_id in the invoice_items table, establishing a one-to-many relationship between invoices and invoice_items.
- The treatments table has a primary key column id and is referenced as a foreign key treatment_id in the invoice_items and joined tables, establishing a many-to-many relationship between treatments and invoices through the invoice_items table and a many-to-many relationship between treatments and medical_histories through the joined table.
- The invoice_items table has a primary key column id and references foreign keys invoice_id and treatment_id to establish relationships with invoices and treatments respectively.

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @chepkok3 @amklass11, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that was implemented on the `dev` branch. Check the screenshot below.
image
The commits should be made on the feature branch and seeing as this project was approved in the previous review, you are expected to start doing this project afresh on a new branch

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    id INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    type varchar(20) NOT NULL,
+    name varchar(50) NOT NULL
+);
+
+-- create invoice items table
+CREATE TABLE invoice_items (
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    unit_price DECIMAL NOT NULL,
+    quantity INT NOT NULL,
```
Comment (Ghiftee) [inline schema_based_on_diagram.sql:38]: - [ ] You are currently missing `total_price` from the invoice_items table. Kindly look at the ERD and add it.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @chepkok3 and @amklass11  👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Proper Gitflow is used
✔️ FKs have indexes created
✔️ Tables are created matching the ERD

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (chepkok3) [reply schema_based_on_diagram.sql:38]: @Ghiftee Okay. I will check it out

Comment (chepkok3) [reply schema_based_on_diagram.sql:38]: @Ghiftee Done.

**Comment by arslanbisharat** [comment]
Hi @chepkok3,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**